.github/workflows/release_build.yaml

```diff
@@ -70,56 +106,69 @@ jobs:
           name: ltheory-bin-ubuntu2004
           path: ${{github.workspace}}/bin/*
 
-  release:
-    name: Update 'latest' release
-    if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+  tag:
+    name: New tag
     needs:
+      - gen-build-version
       - build-windows
       - build-macos
       - build-ubuntu
     runs-on: ubuntu-latest
     steps:
-      - name: Download artifacts
-        uses: actions/download-artifact@v3
-      - name: Inspect directory after downloading artifacts
-        run: ls -alFR
-      - name: Zip artifacts
-        run: for dir in $(ls); do cd $dir && zip ../$dir.zip ./* -r && cd ..; done
       - name: Get latest release ID
         uses: octokit/request-action@v2.x
         id: get_latest_release
         with:
-          route: GET /repos/Limit-Theory-Redux/ltheory/releases/tags/latest
+          route: GET /repos/${{github.repository}}/releases/tags/latest
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      - name: Delete latest release
+      - name: Delete 'latest' tag ref
         uses: octokit/request-action@v2.x
         with:
-          route: DELETE /repos/Limit-Theory-Redux/ltheory/releases/{release_id}
-          release_id: ${{ fromJson(steps.get_latest_release.outputs.data).id }}
+          route: DELETE /repos/${{github.repository}}/git/refs/tags/latest
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      - name: Delete 'latest' tag
+      - name: Delete latest release
```

Comment:
How come you swapped deleting the release and the tag here?

Reply:
accidental :D